### ISSUE TITLE
Not returning artist or song name of Youtube video (Python)

### ISSUE THREAD

**Issue by ChoiYun** [blackjack4494/yt-dlc#49]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dlc version **2020.10.26**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
[youtube] P5ZJui3aPoQ: Downloading webpage
Track: None
Artist: None
```

## Description

When using the **YoutubeDL.extract_info**, that returns **None** to both **track** and **artist** (it returns None to more things like **album, release_date, release_year, etc**).

As described in the solution [here](https://www.reddit.com/r/youtubedl/comments/hyys57/youtube_dl_not_returning_artist_or_song_name/), apparently changing the User-Agent to the Facebook's one fixed the problems, but, for me, even changing stills returning None.

I believe it's a regex problem while scrapping the metadata, but don't have a clue how/where to fix.

Even videos that have this information given in the description, as [Carry on Wayward Son](https://www.youtube.com/watch?v=P5ZJui3aPoQ) - https://imgur.com/55N5k4l, it stills returning None to the **YoutubeDL.extract_info**.

**Python Example for [Carry on Wayward Son](https://www.youtube.com/watch?v=P5ZJui3aPoQ)**
```
YTDL_OPTIONS = {
        'ignoreerrors': False,
        'logtostderr': False,
        'quiet': False,
        'no_warnings': False,
    }

ytdl = ytb.YoutubeDL(YTDL_OPTIONS)

example_url = "https://www.youtube.com/watch?v=P5ZJui3aPoQ"
video = ytdl.extract_info(example_url, download=False)

print('Track: ' + video['track'])
print('Artist: ' + video['artist'])
```

**Output**
```
[youtube] P5ZJui3aPoQ: Downloading webpage
Track: None
Artist: None
```

Anyway, it seems to works well with videos that have the description generated by the Youtube, like this [one](https://www.youtube.com/watch?v=QBxSQXbj6Go)

**Python Example for the link above**
```
YTDL_OPTIONS = {
        'ignoreerrors': False,
        'logtostderr': False,
        'quiet': False,
        'no_warnings': False,
    }

ytdl = ytb.YoutubeDL(YTDL_OPTIONS)

example_url = "https://www.youtube.com/watch?v=QBxSQXbj6Go"
video = ytdl.extract_info(example_url, download=False)

print('Track: ' + video['track'])
print('Artist: ' + video['artist'])
```

**Output**
```
[youtube] QBxSQXbj6Go: Downloading webpage
Track: Keep Pushin'
Artist: REO Speedwagon
```

**Comment by userlip** [comment]
There is no Track and Artist info given in this video.

Check the bottom of the description of this Video to see how it looks like when the video provides that info.
https://www.youtube.com/watch?v=zpOULjyy-n8

**Comment by ChoiYun** [comment]
> There is no Track and Artist info given in this video.
> 
> Check the bottom of the description of this Video to see how it looks like when the video provides that info.
> https://www.youtube.com/watch?v=zpOULjyy-n8

As you can see in the verbose log, the ID of the video that I tested that returned None is the video [Rick Astley - Never Gonna Give You Up](https://www.youtube.com/watch?v=dQw4w9WgXcQ), and in the description is given the artist name and track name. Even this one that you replyed stills returning None, just replace it in the code that I given and see yourself.

I'll edit my issue to make it more clear that I used a video that has that information in the description.
![rick](https://user-images.githubusercontent.com/68434686/97590830-ce078c00-19dd-11eb-8012-bc94251b0d57.png)

**Comment by ChoiYun** [comment]
Fixed by @insaneracist on #55
